x-pack/plugins/ingest_manager/common/types/models/datasource.ts

```diff
@@ -43,6 +43,9 @@ export interface DatasourceInput extends Omit<NewDatasourceInput, 'streams'> {
   streams: DatasourceInputStream[];
 }
 
+/**
+ * Type of `datasource` prop in CustomConfigureDatasourceContent
```

Comment:
Not sure about this comment.
`NewDatasource` is the interface needed to supplement the create API for datasource and I think is reused on the server as well - so its not only a client side UI type.

Reply:
ok i might just take that comment out